loadPluginsToTempDir.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs');
+const _  = require('underscore');
+require('dotenv').config()
+
+const directoryPath = './src/pluginsTemp'
+const separator = require('path').sep
+
+
+const copyFile = (plugin) => {
+  console.log('copying', plugin.path, 'to', plugin.newPath)
+  fs.copyFileSync(plugin.path, plugin.newPath)
+}
+
+const handleError = (error) => {
+  console.log(error)
+}
+
+const hydratePlugin = (plugin) => {
+  var filename = _.last(plugin.path.split(separator))
+
+  plugin.newPath = [directoryPath, filename].join(separator)
+
+  return plugin
+}
+
+const plugins = (environment = process.env) => {
+  if ( !environment.PLUGINS ) { console.log('No plugins are defined.'); return }
+
+  return _.map(JSON.parse(environment.PLUGINS), hydratePlugin)
+}
+
+const setup = () => {
+  if (fs.existsSync(directoryPath))
+    console.log('A temporary plugins folder already exists.')
+  else {
+    console.log('Creating temp plugin folder.')
+    fs.mkdir(directoryPath, handleError)
+  }
+
+  _.each(plugins(), copyFile)
+}
+
+const teardown = () => {
+  // Not triggered by ctrl-C. :(
+  console.log('Removing temp plugin folder.')
+  fs.rmdir(directoryPath, handleError)
+}
+
+if (process.argv.includes('--setup')) {
```

Comment:
Call teardown() here before setup()? That would remove any 'pollution' of the directory by previous setup that wasn't accompanied by a teardown (such as with ctrl-c on the node cli or (I assume) power loss to the node.